build.gradle.kts

```diff
@@ -11,18 +11,19 @@ import java.util.zip.ZipOutputStream
 import org.jetbrains.intellij.tasks.RunPluginVerifierTask
 import org.jfrog.gradle.plugin.artifactory.dsl.PublisherConfig
 
+@Suppress("DSL_SCOPE_VIOLATION")
 plugins {
-    kotlin("jvm") version "1.8.10"
-    id("org.jetbrains.intellij") version "1.15.0"
-    id("org.sonarqube") version "4.4.1.3373"
+    kotlin("jvm") version "1.9.22"
```

Comment:
Out of curiosity, why is this one hardcoded?

Reply:
Right, I forgot it's just the ID `org.jetbrains.kotlin.jvm` behind